apolloschurchapp/src/tabs/index.js

```diff
@@ -39,6 +41,17 @@ SearchButton.propTypes = {
   onPress: PropTypes.func,
 };
 
+const Avatar = withTheme(({ theme: { sizing: { baseUnit } } }) => ({
+  size: 'small',
+  containerStyle: {
+    bottom: baseUnit * 0.25,
+  },
+}))(UserAvatarConnected);
+
+const HeaderLeft = () => {
+  const navigation = useNavigation();
+  return <Avatar onPressIcon={() => navigation.navigate('Connect')} />;
```

Comment:
Curiously, this doesn't take you to the Connect Screen yet. It seems like it should work because it is written the same as in the HeaderRight arrow function in line 58. I event changed Search to Connect in line 58 and it works, but here it doesn't. UserAvatarConnected has an onPressIcon prop instead of onPress. Also, I tried wrapping the avatar in a touchable component with an onPress, but no luck there either. I feel like this may be a navigation issue, but I'm not sure why. Do you see anything that I'm missing?

Reply:
Also, I had the Connect screen setup in the src>index.js file right below Search, but have removed it for the time being.